float32 angle_x
float32 angle_y
float32 angle2north
float64 lat
float64 lon
float64 utm_x
float64 utm_y
int8 quality
float32 speed

